pages/common/pssh.md

```diff
@@ -1,27 +1,27 @@
 # pssh
 
-> Parallel ssh program.
+> Parallel SSH program.
 
-- Run a command on host1 and host2 to print the date on each server inline:
+- Run a command on two hosts, and print its output on each server inline:
 
-`pssh -i -H "{{host1}} {{host2}}" {{date}}`
+`pssh -i -H "{{host1}} {{host2}}" {{echo "foo"}}`
 
 - Run a command and save the output to separate files:
 
-`pssh -H {{host1}} -H {{host2}} -o {{path/to/output_dir}} {{date}}`
+`pssh -H {{host1}} -H {{host2}} -o {{path/to/output_dir}} {{echo "foo"}}`
 
-- Run a command on each host specified in a new-line separated host file:
+- Run a command on multiple hosts, specified in a new-line separated file:
 
-`pssh -i -h {{path/to/hosts_file}} {{date}}`
+`pssh -i -h {{path/to/hosts_file}} {{echo "foo"}}`
 
-- Run a command as root; asks for root password:
+- Run a command as root (this asks for the root password):
 
-`pssh -i -h {{path/to/hosts_file}} -A -l {{root}} {{date}}`
+`pssh -i -h {{path/to/hosts_file}} -A -l {{root_username}} {{echo "foo"}}`
```

Comment:
Why `root_username` annd not `username` or just `root`?

Reply:
Just `root` (what it was previously) is ambiguous IMO (is it a username, the word "root", something else?). Just `username` wouldn't make it sufficiently clear that the user must be root. I think the expanded token strikes an acceptable balance between explicitness and conciseness.

Reply:
Ah, cool :D